### PR TITLE
Bumping otel version to v0.18.0. Prepare for releasing v0.18.0

### PR DESCRIPTION
**Pull request by Aneurysm9**
### Fixed

- `otelmemcache` no longer sets span status to OK instead of leaving it unset. (#477)
- Fix goroutine leak in gRPC `StreamClientInterceptor` (#581)

### Removed

- Remove service name from `otelmemcache` configuration and span attributes. (#477)

### CODE REVIEW

File: exporters/metric/datadog/example_test.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func ExampleExporter() {
 	}
 
 	// Output:
-	// myrecorder.max:100|g|#env:dev,l:1
+	// myrecorder.max:100|g|#env:dev,l:1,service.name:unknown_service:datadog.test,telemetry.sdk.language:go,telemetry.sdk.name:opentelemetry,telemetry.sdk.version:0.18.0
```
Comment (MrAlias) [inline exporters/metric/datadog/example_test.go:101]: This is going to be tricky. Every upgrade this test will need to be updated.

File: instrumentation/github.com/Shopify/sarama/otelsarama/message.go [hunk 2]
```diff
@@ -58,6 +58,15 @@ func (c ProducerMessageCarrier) Set(key, val string) {
 	})
 }
 
+// Keys returns a list of all key identifiers in the carrier.
```
Comment (MrAlias) [inline instrumentation/github.com/Shopify/sarama/otelsarama/message.go:61]: nit

```suggestion
// Keys returns a slice of all key identifiers in the carrier.
```

File: instrumentation/github.com/Shopify/sarama/otelsarama/message.go [hunk 3]
```diff
@@ -92,3 +101,12 @@ func (c ConsumerMessageCarrier) Set(key, val string) {
 		Value: []byte(val),
 	})
 }
+
+// Keys returns a list of all key identifiers in the carrier.
```
Comment (MrAlias) [inline instrumentation/github.com/Shopify/sarama/otelsarama/message.go:105]: nit

```suggestion
// Keys returns a slice of all key identifiers in the carrier.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,9 +8,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [0.18.0] - 2021-03-04
+
 ### Fixed
 
 - `otelmemcache` no longer sets span status to OK instead of leaving it unset. (#477)
+- Fix goroutine leak in gRPC `StreamClientInterceptor` (#581)
```
Comment (MrAlias) [inline CHANGELOG.md:16]: nit

```suggestion
- Fix goroutine leak in gRPC `StreamClientInterceptor`. (#581)
```

File: instrumentation/github.com/emicklei/go-restful/otelrestful/example/server.go [hunk 5]
```diff
@@ -23,8 +23,8 @@ import (
 
 	"go.opentelemetry.io/contrib/instrumentation/github.com/emicklei/go-restful/otelrestful"
 	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/exporters/stdout"
-	otelkv "go.opentelemetry.io/otel/label"
```
Comment (MrAlias) [inline instrumentation/github.com/emicklei/go-restful/otelrestful/example/server.go:27]: talk about a relic :smile:

**Review by MrAlias — APPROVED** [review]
:1st_place_medal:!

File: instrumentation/github.com/Shopify/sarama/otelsarama/message.go (hunk 2 above)
Comment (Aneurysm9) [reply instrumentation/github.com/Shopify/sarama/otelsarama/message.go:61]: My Perl is showing!

File: exporters/metric/datadog/example_test.go (hunk 1 above)
Comment (Aneurysm9) [reply exporters/metric/datadog/example_test.go:101]: Yeah, though if it's just the version identifier we might be able to work that into `pre_release.sh` and automate it.

**Review by MrAlias — APPROVED** [review]

**Pull request merged**